internal/api/health.go

```diff
@@ -0,0 +1,69 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/pass-wall/passwall-server/internal/config"
+	"github.com/pass-wall/passwall-server/internal/storage"
+)
+
+var (
+	// should be improved
+	Port          = config.SetupConfigDefaults().Server.Port
```

Comment:
I think, it is NOT proper way of getting port from configuration. could be improved. (Maybe organization of packages may change )

Reply:
You are right, it also bothers my eye. I will also think about it.

Reply:
Could be a static variable for test purposes. 

```suggestion
	Port          = '3625'
```